src/components/Comments/Comment.js

```diff
@@ -46,6 +55,15 @@ const Comment = ({
     level,
     updateVote
   });
+  const [loading, setLoading] = React.useState({
+    loadingDm: false,
+    loadingDmAnon: false
+  });
+
+  const [profile] = React.useContext(Context).profile;
+  const {createSignChat} = useCreateChat();
+  const {sendMessageDM} = useDMMessage();
+
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

You are initializing `loading` state with an object that has two properties: `loadingDm` and `loadingDmAnon`. This is fine, but it might be more readable and maintainable to separate these into two different state variables. This way, you can avoid having to spread the previous state every time you update one of the properties.
```diff
- const [loading, setLoading] = React.useState({
-   loadingDm: false,
-   loadingDmAnon: false
- });
+ const [loadingDm, setLoadingDm] = React.useState(false);
+ const [loadingDmAnon, setLoadingDmAnon] = React.useState(false);
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->